src/util/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+add_library(clio_util)
+
+target_sources(
+  clio_util
+  PRIVATE config/Config.cpp
+          log/Logger.cpp
+          prometheus/Http.cpp
+          prometheus/Label.cpp
+          prometheus/MetricBase.cpp
+          prometheus/MetricBuilder.cpp
+          prometheus/MetricsFamily.cpp
+          prometheus/OStream.cpp
+          prometheus/Prometheus.cpp
+          Random.cpp
+          Retry.cpp
+          requests/RequestBuilder.cpp
+          requests/Types.cpp
+          requests/WsConnection.cpp
+          requests/impl/SslContext.cpp
+          Taggable.cpp
+          TerminationHandler.cpp
+          TxUtils.cpp
+          LedgerUtils.cpp
+)
+
+target_link_libraries(
+  clio_util
+  PUBLIC Boost::headers
+         Boost::stacktrace_backtrace
+         dl
+         libbacktrace::libbacktrace
+         fmt::fmt
+         openssl::openssl
+         xrpl::libxrpl
+         Threads::Threads
+         clio_options
+)
+
+if (is_gcc)
+  # FIXME: needed on gcc for now For some reason cmake doesn't propagate the compile definitions from clio_options so we
```

Comment:
Please split the comment into two lines "now\nFor some".

Reply:
Hm, it was on different lines until cmake-format merged it 🤔

Reply:
Perhaps it wants a `.` at the end of the sentence? :rage2: